src/content/docs/en/guides/integrations-guide/mdx.mdx

```diff
@@ -151,7 +153,6 @@ See more examples of using import and export statements in the [MDX docs](https:
 
 ```mdx title="src/pages/about.mdx" {5-6} /<.+\/>/
 ---
-layout: ../layouts/BaseLayout.astro
```

Comment:
One issue and a question about this code block.

**Issue:** Since `layout` is removed, the highlighted lines should be updated to `{4-5}` I think.

**Question:** The path is `src/pages/about.mdx` so shouldn't we update the code to add a `<Layout />` component? Somethink like:

````
```mdx title="src/pages/about.mdx" {4-7} /<.+\/>/
---
title: About me
---
import Button from '../components/Button.astro';
import Layout from '../components/Layout.astro';
import ReactCounter from '../components/ReactCounter.jsx';

<Layout>
I live on **Mars** but feel free to <Button title="Contact me" />.

Here is my counter component, working in MDX:

<ReactCounter client:load />
</Layout>
```
````
It might also be better if we want to encourage the use of a `<Layout>` component rather than the `layout` property.

Reply:
Ah, thanks for catching the line highlighting!

In fact, I'd rather update the file path than add a layout, since that's how we want to encourage most use anyway. And, I always like it when code examples get shorter, not longer! :joy: So let's make this look like something in a collection instead.

What about...

``` title="src/blog/post-1.mdx"
---
title: My first post
---
import ReactCounter from '../components/ReactCounter.jsx';

I just started my new Astro blog! 

Here is my counter component, working in MDX:
<ReactCounter client:load />
```

Reply:
> And, I always like it when code examples get shorter, not longer! 😂 

I suspected that! 😆  I had hesitated to just rename the path... 
 
> What about...

Looks good to me! But maybe we should keep the highlighting for `import ReactCounter...` and `<ReactCounter ...`?

Reply:
Yes, I was just being lazy. We can add the proper highlighting for that! If you make that update, we can commit then merge it!